frontend/packages/dev-console/src/utils/application-utils.ts

```diff
@@ -194,6 +226,82 @@ const updateItemAppConnectTo = (
   return k8sPatch(model, item, patch);
 };
 
+export const createServiceBinding = (
+  source: K8sResourceKind,
+  target: K8sResourceKind,
+): Promise<any> => {
+  if (!source || !target || source === target) {
+    return Promise.reject();
+  }
+
+  const targetName = _.get(target, 'metadata.name');
+  const sourceName = _.get(source, 'metadata.name');
+  const namespace = _.get(source, 'metadata.namespace');
+  const targetResourceGroup = _.split(_.get(target, 'metadata.ownerReferences[0].apiVersion'), '/');
+  const targetResourceKind = _.get(target, 'metadata.ownerReferences[0].kind');
+  const targetResourceRefName = _.get(target, 'metadata.ownerReferences[0].name');
+  const sbrName = `${sourceName}-${source.kind}-${targetName}-${target.kind}`;
+
+  const serviceBindingRequest = {
+    apiVersion: 'apps.openshift.io/v1alpha1',
+    kind: 'ServiceBindingRequest',
+    metadata: {
+      name: sbrName,
+      namespace,
+    },
+    spec: {
+      applicationSelector: {
+        resourceRef: sourceName,
+        group: 'apps.openshift.io',
```

Comment:
This is the wrong group for a deployment.
This field needs to be dynamic based on the source resource.

Reply:
Done